bigquery/spec/quickstart_spec.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "google/cloud"
+
+describe "BigQuery Quickstart" do
+
+  it "creates a new dataset" do
+    # Initialize test objects
+    gcloud_test_client = Google::Cloud.new ENV["GOOGLE_CLOUD_PROJECT"]
```

Comment:
Let's change these to just `gcloud` and `bigquery` like is used everywhere else.

I liked how explicit `gcloud_test_client` is when initially drafting these specs to be super clear and self-document how these specs work:
https://gist.github.com/remi/f3abdee163a20e56c4d678c10e088d09

But now that I'm going back and reading these... I'd like to use the simple, consistent `gcloud` and `project_name` variables for these)

#consistency #nit